spec/controllers/tasks_controller_spec.rb

```diff
@@ -9,7 +10,6 @@
   let(:collection) { create(:collection, users: [user], tasks: []) }
   let(:valid_attributes) { {user:, access_level:} }
   let(:access_level) { :private }
-
```

Comment:
This change seems unrelated, is it?

Reply:
not sure what you are referring to here?

Reply:
I probably just removed an empty space I think. that's all. Is that what you are talking about?